### DISCUSSION TITLE
cannot reverify academic affiliation

### DISCUSSION THREAD

**Discussion by markov42** [community/community#51983] [GitHub Education]
Hi,

I am having trouble reverifying my academic affiliation. What picture exactly should I take to reverify? I uploaded a picture of my canvas page and a picture of my I-20, a picture of my student ID, but none of them work. And it requires me to be physically near my campus right now, but I am out of state doing my internship until the end of June. Could you please help me with this verification?

Thanks,
Jing

**Comment by samus-aran** [comment]
Hi @markov42 

Lets see how we can assist ⭐ 

For remote student applicants: please ensure that your evidence indicates that you are a remote student. If possible, we encourage students to apply whilst on campus. This will assist us in confirming your student status.

Also include evidence that has a date to verify the length of your academic studies. You can have multiple evidence in the same photo if possible. 

Let us know how you get on 👍
